src/domain/event/similarEvents/__tests__/SimilarEvents.test.tsx

```diff
@@ -83,3 +98,30 @@ test('should render similar event cards', async () => {
     ).toBeInTheDocument();
   });
 });
+
+it('has return path on similar event link', async () => {
+  const path = ROUTES.EVENT;
+  const route = path.replace(':id', event.id);
+  const { history } = await waitForComponentToBeLoaded(() =>
+    renderWithRoute(<SimilarEvents event={event} />, {
+      mocks,
+      path,
+      routes: [route],
+    })
+  );
```

Comment:
not sure why `renderWithRoute` is used in this test scenario. I think this is meant to be used with components that are actual route components (top level page component)

this works also here
```
 const { history } = await waitForComponentToBeLoaded(() =>
    render(<SimilarEvents event={event} />, {
      mocks,
      routes: [route],
    })
  );
```

Reply:
Of course a comment could be added to renderWithRoute utility to clarify this

Reply:
That's strange, it did not work previously with `render` but now it does. I thought that the router is needed for the history operation. But I did originally copy-cat this from https://github.com/City-of-Helsinki/events-helsinki-ui/blob/master/src/domain/event/eventCard/__tests__/EventCard.test.tsx#L88-L125

Reply:
I need to check if EventCard.test.tsx needs renderWithRoute

Reply:
seems like EventCard test needs renderWithRoute to get this hook return params:

`  const params = useParams<{ place?: string }>();`